.travis.yml

```diff
@@ -33,7 +33,7 @@ matrix:
       before_install: echo "Going to run style checks using pre-commit"
       install:
         - pip install pre-commit
-        - pre-commit install
+        - pre-commit install --install-hooks
```

Comment:
More curiosity than anything else but what's the relevance of this option? The documentation isn't very clear: "Whether to install hook environment for all environments in the config file;"

Reply:
Having watched this with and without a pre-existing cache, it controls when the environments are created.

```
pip install pre-commit
pre-commit install
pre-commit run -a # environments created now
```

versus:

```
pip install pre-commit
pre-commit install --install-hooks  # environments created now
pre-commit run -a
```

With the way travis splits things up and times them, it seemed nicer to have this as part of the install step, not the main script. Also it makes the main output less verbose (unless the environments are cached already).